src/core/graph.js

```diff
@@ -432,6 +432,13 @@ export class Graph {
     return result;
   }
 
+  /**
```

Comment:
Please remove this method, as it's not actually needed for this PR (since the timelineCred and the graph may have different sets of nodes).

right now, across the codebase, when we do need node addresses from the graph, we do `Array.from(graph.nodes()).map((x) => x.address)`. if we were to add a node address method to the graph, we would want it in its own PR where we refactor all the calls of that form to use the new method. Also, the node address method would need to return the node addresses in sorted order (like the other nodes methods on the graph) rather than the insertion order of the map.